pkg/vault/test/foundry/E2eErc4626Swaps.t.sol

```diff
@@ -329,12 +317,14 @@ contract E2eErc4626SwapsTest is BaseERC4626BufferTest {
         uint256 senderDaiDelta = balancesBefore.balances.bobTokens[balancesBefore.daiIdx] -
             balancesAfter.balances.bobTokens[balancesAfter.daiIdx];
         uint256 vaultTotalDaiBefore = balancesBefore.balances.vaultTokens[balancesBefore.daiIdx] +
-            waDAI.previewRedeem(balancesBefore.balances.vaultTokens[balancesBefore.waDaiIdx]);
+            waDAI.previewMint(balancesBefore.balances.vaultTokens[balancesBefore.waDaiIdx]);
         uint256 vaultTotalDaiAfter = balancesAfter.balances.vaultTokens[balancesAfter.daiIdx] +
-            waDAI.previewRedeem(balancesAfter.balances.vaultTokens[balancesAfter.waDaiIdx]);
+            waDAI.previewMint(balancesAfter.balances.vaultTokens[balancesAfter.waDaiIdx]);
```

Comment:
This just changes the rounding to up; does it make a difference in this case? Can you explain the rationale?

Reply:
No difference at all. I changed it to test a few things and didn't change it back.